Makefile

```diff
@@ -49,5 +49,8 @@ obj/ActiveInventory.o: src/ActiveInventory.cpp
 obj/InventoryManager.o: src/InventoryManager.cpp obj/ActiveInventory.o
 	$(CC) $(CFLAGS) -c src/InventoryManager.cpp -o obj/InventoryManager.o
 
+obj/Sales.o: src/Sales.cpp obj/Sales.o
```

Comment:
This is a circular dependency, obj/Sales.o is dependent on obj/Sales.o.
It should be:
```obj/Sales.o: src/Sales.cpp```